src/lib/fixtures/dockerfileContent.ts

```diff
@@ -1,22 +1,26 @@
 export const dockerfileContent = `FROM node:18.16.1-alpine
+# Use an official Node.js runtime as a parent image 
 
-RUN apk add --no-cache bash
-RUN npm i -g @nestjs/cli typescript ts-node
+# Set the working directory in the container
+WORKDIR /usr/src/app
 
-COPY package*.json /tmp/app/
-RUN cd /tmp/app && npm install
+# Copy the package.json and package-lock.json files to the container
+COPY package*.json ./
 
-COPY . /usr/src/app
-RUN cp -a /tmp/app/node_modules /usr/src/app
-COPY ./wait-for-it.sh /opt/wait-for-it.sh
-COPY ./startup.dev.sh /opt/startup.dev.sh
-RUN sed -i 's/\r//g' /opt/wait-for-it.sh
-RUN sed -i 's/\r//g' /opt/startup.dev.sh
+RUN npm i -g @nestjs/cli @samagra-x/stencil-cli typescript ts-node
```

Comment:
don't install both clis it will cause a lot of conflicts in commands

Reply:
okay, so ill just keep @samagra-x/stnecil-cli ?

Reply:
Yes